### PR TITLE
Add automation standards for PowerShell scripts

### PR DESCRIPTION
**Pull request by tksh164**
# Overview/Summary

Added the automation standards for PowerShell scripts to the Contribution Guide.

## Related Issues/Work Items

- #213

## This PR fixes/adds/changes/removes

1. Adds the automation standards for PowerShell scripts to the Contribution Guide.

### Breaking Changes

- None

## As part of this Pull Request I have

- [x] Read the [Contribution Guide](https://azure.github.io/Azure-Proactive-Resiliency-Library/contributing) and ensured this PR is compliant with the guide
- [x] Checked for duplicate [Pull Requests](https://github.com/Azure/Azure-Proactive-Resiliency-Library/pulls)
- [x] Associated it with relevant [GitHub Issues](https://github.com/Azure/Azure-Proactive-Resiliency-Library/issues) or ADO Work Items (Internal Only)
- [x] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Azure-Proactive-Resiliency-Library/tree/main)
- [x] Ensured PR tests are passing
    - Confirmed complete document build successfully on my local machine.
- [ ] Performed testing and provided evidence (e.g. screenshot of output) for any changes associated to ARG queries and/or scripts
- [x] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

File: docs/content/contributing/_index.md [hunk 1]
```diff
@@ -199,6 +199,57 @@ NOTE: The column names should be in the order they are listed and match exactly.
...
+
+    ```powershell
+    // Azure PowerShell script
+    // Provides a list of Azure Container Registry resources that do not have soft delete enabled
+    ```
+
+1. Scripts should only return resources that do not adhere to the APRL recommendation. For example, if the recommendation is to enable soft delete for Azure Container Registries, the associated scripts should only return Azure Container Registry resources that do not have soft delete enabled.
+
+1. Scripts should only include codes to return resources that do not adhere to the APRL recommendation. Scripts should not include the support code such as sign-in to Azure ([Connect-AzAccount](https://learn.microsoft.com/en-us/powershell/module/az.accounts/connect-azaccount), Login-AzAccount), subscription selection ([Set-AzContext](https://learn.microsoft.com/en-us/powershell/module/az.accounts/set-azcontext), Select-AzSubscription), etc. Those codes should be executed outside of the APRL recommendation PowerShell script.
```
Comment (oZakari) [inline docs/content/contributing/_index.md:213]:
```suggestion
1. Scripts should exclusively contain code to retrieve resources that do not comply with the APRL recommendation. They should not include supporting code, such as Azure sign-in ([Connect-AzAccount](https://learn.microsoft.com/en-us/powershell/module/az.accounts/connect-azaccount), Login-AzAccount) or subscription selection ([Set-AzContext](https://learn.microsoft.com/en-us/powershell/module/az.accounts/set-azcontext), Select-AzSubscription). Execute these codes separately from the APRL recommendation PowerShell script.

File: docs/content/contributing/_index.md [hunk 2]
```diff
@@ -199,6 +199,57 @@ NOTE: The column names should be in the order they are listed and match exactly.
...
+
+1. The script should returned the result as an array of `PSCustomObject` data type, and each result object should only include the following properties:
```
Comment (oZakari) [inline docs/content/contributing/_index.md:215]:
```suggestion
1. The script should return the result as an array of the `PSCustomObject` data type, with each result object containing only the following properties:
```

**Review by oZakari — CHANGES_REQUESTED** [review]
Thanks @tksh164! Please review my suggestions and let me know if you have any questions.

**Comment by tksh164** [comment]
@oZakari - Thanks for touching up my text!

**Review by oZakari — APPROVED** [review]

**Review by ejhenry — APPROVED** [review]
LGTM, thanks @tksh164 and @oZakari!

**Pull request merged**